### ISSUE TITLE
Can Not Resize Columns

### ISSUE THREAD

**Issue by LookingForWayOut** [lkosson/full-address-column#13]
Hello. My Thunderbird is updated to 102.6.1 (64b) on W11 Pro.
I am not sure why but I can not resize the columns anymore. Right now me recipients column is too long and the sender is only a few characters long. I can not change either field.
Any thoughts?
Thank you.

**Comment by lkosson** [comment]
I can reproduce the behavior using the same TB version on W10 and W7. It happens, when the add-on column is next to a built-in column with fixed width, such as Spam column or a show/hide columns button (last column). Curiously, it affects other columns as well, with add-on disabled.

For a workaround, move column to some other location (between normal columns), resize it there, and move it back.

**Comment by LookingForWayOut** [comment]
That worked.

I was moving columns around the other day thinking the same thing, but not the way you mentioned. I only have (in position order left to right) **date, favorite, attachment, read, subject, SENDER/RECIPIENT, size.** When I tried moving them around, I only put them before **subject** or after **size** and that did not work. As you suggested, I moved them between **read** and **subject** and it worked. I was able to resize them accordingly and put them back in position. Took a few minutes tog et the size right because resize one column affects the others, but I got it set. And yes, where they are now, I can not resize the **size** column at the end unless I move that far left as well.

Thank you. My OCD is happy again for the moment.
